.engine_scripts/playwright/actions.js

```diff
@@ -85,10 +85,10 @@ module.exports = async (currentPage, scenario) => {
 
         files.forEach((file) => {
           if (path.isAbsolute(file)) {
-            throw '`file` must be relative path to "data" folder';
+            throw '`file` must be relative path to "visual_test" folder';
```

Comment:
Please change the name to `visual-tests` (with `s`) to indicate that it can contains more than one file.

Reply:
hi @tuyen-at-work i updated it & add github action auto deploy to npm. Pls help me review again. Thanks